lib/OpenQA/CLI/clone.pm

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::CLI::clone;
+use Mojo::Base 'OpenQA::Command';
+
+use Mojo::Util qw(getopt);
+use OpenQA::Script::CloneJob;
+
+has description => 'Clone a job from a local or remote instance';
+has usage       => sub { shift->extract_usage };
+
+sub command {
+    my ($self, @args) = @_;
+
+    my $from;
+    getopt \@args,
+      'from=s'   => sub { $from = $_[1] },
+      'from-o3'  => sub { $from = 'https://openqa.opensuse.org' },
+      'from-osd' => sub { $from = 'http://openqa.suse.de' },
```

Comment:
Adding just the from options without also adding the corresponding within-instance as well as to parameters can be seen inconsistent. Adding all of that might be a bit too much. Maybe better shortcuts could be to define aliases for host values themselves? E.g. so that we could write `--from o3 --to osd`

Reply:
You mean `--from o3 --host osd`? Would that apply to `--within-instance`, too? And to other `cli` commands? When I previously suggested it, it didn't seem like anyone else was in favor of that approach.